packages/inter-protocol/test/smartWallet/test-oracle-integration.js

```diff
@@ -149,7 +149,7 @@ const acceptInvitation = async (wallet, priceAggregator) => {
 
 let pushPriceCounter = 0;
 /**
- * @param {*} wallet
+ * @param {import('@agoric/smart-wallet/src/smartWallet.js').SmartWallet} wallet
```

Comment:
MH usually avoids changes that have no runtime impact on the release branch. I'm not sure whether that matters here.

Reply:
not a problem in test files